data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1875 @@ mission "Stone of our Fathers 5"
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label noquestion
+			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
+			`	"Enough ice-plain, now ve go to next stop."`
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label rock
+			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`
+			`	"This is rock." She says matter-of-factly.`
+			`	You look at it for a while and realize there is a face half chiseled into it.`
+			`	"You are done looking? Ve are good to move to next stop?" Hjlod asks.`
+			scene "scene/lonelyrock"
+			choice
+				`	"Yep, I don't need to look at this rock any longer."`
+					goto byerock
+				`	"Wait, is that part of a man's face carved into it? What is so special about this rock?"`
+					goto manrock
+			label manrock
+			`	"That rock is supposed to celebrate man who discover Rigellian cure. He was called Hank, or Henry, or Bill, or something." Hjlod shrugs with evident disdain.`
+			`	"The cure was found here, but back when tundra was melting and Nifel was a little varmer. They say this man discover it and they want to make grand statue for him. But it turn out Nifel not so great place for statue. Beside, no one like statue of liar."`
+			`	Hjlod turns back to her controls and gets the Ice-Crawler moving forward again. "That is enough about stupid rock and stupider man."`
+			`	"Now ve go to river."`
+					goto riverlike
+			label byerock
+			`	"Now ve go to river."`
+					goto riverlike
+			label riverlike
+			choice
+				`	(The large oddly shaped rock falls out of your field of view as the Ice-Crawler turns away from it.)`
+			scene "scene/snowfield"
+					goto river
+			label river
+			`	The uneventful journey to the boiling ice river of Slidr is suddenly interrupted when the Ice-Crawler begins to shake wildly for a few moments, followed by a thunderous boom from somewhere outside.`
+			`	After Hjlod regains control of the crawler, she immediately veers away from the noise. "That vas Nifel-quake, and loud noise was avalanche. This is not good."`
+			`	Despite her calm demeanor she wields the Ice-Crawler with reckless abandon as she tries to outrun the rumbling snow. The machine protests with creaks and whirrs and pops as it goes faster than it has gone for hundreds of years.`
+			`	"I think..." Hjlod starts to say when a mountain of snow crashes into the Ice-Crawler causing it to roll and tumble for over a kilometer.`
+			`	You and Hjlod hold tight to your seat-belts as you're tossed around. Eventually, the crawler rolls to its side and comes to a rest.`
```

Comment:
Obviously she can't outrun an avalanche, but that doesn't mean she wouldn't try to outrun it. I think that is a pretty standard human response. 

Per your suggestion I've edited the wording around the tumble and kilometer stuff to make it sound less damaging.

Reply:
It's a standard human response, it's also an incredibly stupid one and the very first thing that any sort of wilderness survival experience would teach you not to do.

If you lived in this environment it would be drilled into you to get out of the way, to find shelter, to do anything except remain in the path of the avalanche. She's a native, she should know better.

Reply:
I've changed "outrun" to "avoid". 
I think that should satisfy your concerns.